<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddProfileDialog</class>
 <widget class="QDialog" name="AddProfileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>280</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Profile</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="profileNameLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Profile Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="profileNameEdit"/>
   </item>
   <item>
    <widget class="QCheckBox" name="ssuriCheckBox">
     <property name="text">
      <string>Base64 Encoded URI (Optional)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="scanButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Scan QR code on screen and decode it to get the SS URI.</string>
     </property>
     <property name="text">
      <string>Scan QR Code on Screen</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="qrfileButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>From QR Code Image File</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="format">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="ssuriEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>ss://BASE64-ENCODED-STRING-WITHOUT-PADDING</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>profileNameEdit</tabstop>
  <tabstop>ssuriCheckBox</tabstop>
  <tabstop>scanButton</tabstop>
  <tabstop>ssuriEdit</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>addButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>ssuriCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>ssuriEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ssuriCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>scanButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ssuriCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>qrfileButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
